bayesian/api/api_v2.py

```diff
@@ -60,6 +61,44 @@
 metrics = GunicornPrometheusMetrics(api_v2, group_by="endpoint", defaults_prefix=NO_PREFIX)
 
 
+@api_v2.route('/component-vulnerability-analysis/', methods=['POST'])
+@api_v2.route('/component-vulnerability-analysis', methods=['POST'])
```

Comment:
any specific reason for this?

Reply:
following the standered of v2 CA post api.

Reply:
okay, why is it not consistent? one has "/" other does not ?

Reply:
i am unsure of why it was done in our previous ca api.
Lets wait for comment from @yzainee

Reply:
looks like a typo to me. dont see any reason for having it twice

Reply:
for the old api, i think it was related to vscode where the call was coming with or without '/' . dont recall it fully

Reply:
here: https://github.com/fabric8-analytics/fabric8-analytics-server/pull/761

Reply:
@rafiu007 can you check and resolve this

Reply:
ACK .I remeber too now.
will remove the one without /

Reply:
Resolved.
the lash endpoint is lsp specific.We dont need one here.